homeassistant/components/livisi/switch.py

```diff
@@ -169,11 +173,9 @@ def update_states(self, device_id_state) -> None:
     @callback
     def update_reachability(self, device_id_reachability) -> None:
         """Update the reachability of the switch device."""
-        if device_id_reachability is None:
-            return
-        if device_id_reachability.get("id") != self.unique_id:
+        if device_id_reachability[ID] != self.unique_id:
             return
-        if device_id_reachability["is_reachable"] is False:
+        if device_id_reachability[IS_REACHABLE] is False:
```

Comment:
We can assign `self._attr_available` the value of this item as its already a boolean with the same meaning.